Classes/Settings/SettingsViewController.swift

```diff
@@ -84,15 +84,18 @@ GitHubSessionListener {
             case .success(let response):
                 let text: String
                 let color: UIColor
-                switch response.data.status {
-                case .good:
-                    text = NSLocalizedString("Good", comment: "")
+                switch response.data.status.indicator {
+                case .none:
+                    text = response.data.status.description
                     color = Styles.Colors.Green.medium.color
                 case .minor:
-                    text = NSLocalizedString("Minor", comment: "")
+                    text = response.data.status.description
                     color = Styles.Colors.Yellow.medium.color
                 case .major:
-                    text = NSLocalizedString("Major", comment: "")
+                    text = response.data.status.description
+                    color = Styles.Colors.Red.medium.color
+                case .critical:
+                    text = response.data.status.description
                     color = Styles.Colors.Red.medium.color
```

Comment:
Should we introduce / can we use another color to disambiguate between `.major` and `.critical`?